include/big_core_base.h

```diff
@@ -1,4 +1,5 @@
 #pragma once // RV: nicemu to nevadi a naprosta vetsina prekladacu to podporuje https://en.wikipedia.org/wiki/Pragma_once
+             // LB: ja vim, ale je zbytecne mit tohle a jeste include guarda
```

Comment:
// RV: ne tak docela, pragma once to muze urychlit, include guards kvuli zpetne kompatibilite: If you need your program to work on all compilers then you should use #pragma once and include guards both. (https://stackoverflow.com/questions/13339392/should-i-still-use-include-guards-and-pragma-once). Takze nechame oboje.

Reply:
ok